inscriptions/ISic000001.xml

```diff
@@ -204,11 +204,11 @@
                 <listApp><app><note>Text of CIL</note></app></listApp>
             </div>
             <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
```

Comment:
actually, we leave this in as an instruction for future editors, so don't delete it.